tests/RouteCollectionTest.php

```diff
@@ -398,39 +501,42 @@ public function testDeepGrouping(): void
         $collector = new RouteCollection();
         $collector->get('/', new Fixtures\BlankRequestHandler())->bind('home');
 
-        $collector->group('api.')
-            ->prototype()
+        $collector
+            ->group('api.')
                 ->prefix('/api')
-                ->get('/', new Fixtures\BlankRequestHandler())->bind('home')->end()
-                ->get('/ping', new Fixtures\BlankRequestHandler())->bind('ping')->end()
-                ->group('')
-                    ->prototype()
-                        ->scheme('https', 'http')->method(Router::METHOD_CONNECT)->default('hello', 'world')
-                        ->head('hello', new Fixtures\BlankRequestHandler())->bind('hello')->argument('foo', 'hello')->end()
-                    ->end()
+
+                ->routes([
+                    Route::to('/', handler: new Fixtures\BlankRequestHandler())->bind('home'),
```

Comment:
Avoid passing named parameters on this line as it breaks tests in PHP 7.4.

```suggestion
                    Route::to('/', Route::DEFAULT_METHODS, new Fixtures\BlankRequestHandler())->bind('home'),
```